app/controllers/api/revisions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::RevisionsController < Api::ApiController
+  def show
+    revision = current_user.items.find(params[:item_id]).revisions.find(params[:id])
+
+    render json: revision
+  end
+
+  def index
+    begin
+      item = current_user.items.find(params[:item_id])
+    rescue ActiveRecord::RecordNotFound
+      return render json: { error: 'Item not found' }, status: :not_found
```

Comment:
The error format we [typically use](https://github.com/standardnotes/syncing-server/blob/develop/app/controllers/api/auth_controller.rb#L115) (and clients expect) in the API is `{error : { message: '' } }`